### DISCUSSION TITLE
Is there any way to disable maximize/fullscreen on macos

### DISCUSSION THREAD

**Discussion by wizzymore** [wailsapp/wails#3284] [Q&A] [answered]
Is there any way to disable maximize/fullscreen on macos?

Currently I can't find any way to make the window Resizable and to disable the Fullscreen button or atleast make it Maximize to the MaxWidth instead of making the application Fullscreen.

What i want is something like this, disabled fullscreen button and being able to resize the window manually, the menu of this application is Native not custom:
<img width="1067" alt="image" src="https://github.com/wailsapp/wails/assets/25827787/afeba9cb-353e-4398-9fbc-7279825778c1">

**Comment by leaanthony** [comment, marked as answer]
Thanks for getting a PR done to give you what you need! 👍 I just got this mouse. It's amazing 🔥

**Reply by wizzymore** [reply]
Indeed it is, just got it recently too and it's amazing.
Have a great one and thanks for this amazing framework.
